### DISCUSSION TITLE
Combining CSR and SSG

### DISCUSSION THREAD

**Discussion by vencho-mdp** [nuxt/nuxt#17557] [Questions]
Hi! How would one use CSR and SSG in one app? For e.g: SSG for a blog and CSR for an admin dashboard that does not require SEO. In the docs, it states that if you want to use CSR you have to set `ssr: false`, but that affects all the app routes that use prerendering.

**Comment by TheAlexLichter** [comment]
Fully "client-side-loaded" pages are not possible at the moment due to https://github.com/nuxt/framework/issues/5003.

**Reply by vencho-mdp** [reply]
Oh, and when are they planned to be released? In conjunction with the "Hybrid Mode"?
